src/plugins/workspace/public/components/workspace_creator/workspace_form_summary_panel.tsx

```diff
@@ -190,6 +200,12 @@ export const WorkspaceFormSummaryPanel = ({
           )}
         </FieldSummaryItem>
       )}
+      {isPermissionEnabled && (
+        <FieldSummaryItem field={RightSidebarScrollField.PrivacyType}>
+          {privacyType && <EuiText size="xs">{privacyType2TextMap[privacyType].title}</EuiText>}
```

Comment:
Should it actually check `privacyType2TextMap[privacyType]` here?
```suggestion
          {privacyType2TextMap[privacyType] && <EuiText size="xs">{privacyType2TextMap[privacyType].title}</EuiText>}
```

Reply:
From a type-safe perspective, we don't need to perform any checks here. The `privacyType` was defined as the `WorkspacePrivacyItemType` type above. The `privacyType2TextMap` is using `WorkspacePrivacyItemType` as the map key. The `privacyType2TextMap[privacyType]` must exist. Do you think if we should check privacyType2TextMap[privacyType] exists here? It can be updated in the next collaborators' [flyout PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9145).